static/src/javascripts/projects/common/modules/commercial/third-party-tags.js

```diff
@@ -11,19 +11,19 @@ define([
     'common/modules/commercial/third-party-tags/imr-worldwide',
     'common/modules/commercial/third-party-tags/remarketing',
     'common/modules/commercial/third-party-tags/krux',
-    'common/modules/commercial/third-party-tags/outbrain'
-], function (
-    Promise,
-    config,
-    mediator,
-    commercialFeatures,
-    audienceScienceGateway,
-    audienceSciencePql,
-    imrWorldwide,
-    remarketing,
-    krux,
-    outbrain
-) {
+    'common/modules/commercial/third-party-tags/outbrain',
+    'common/modules/commercial/third-party-tags/plista'
+], function (Promise,
```

Comment:
Most modules in Frontend have their dependencies written in the original file's style.